image/Dockerfile.engine-dm

```diff
@@ -0,0 +1,78 @@
+# Common builder
+ARG GO_IMAGE
+FROM ${GO_IMAGE} as golang
+
+FROM centos:7 as builder
+ENV GOPATH=/go
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+ENV AUTO_GOPATH 1
+COPY --from=golang /usr/local/go /usr/local/go
+COPY hack/dockerfile/install/tini.installer /
+COPY hack/dockerfile/install/proxy.installer /
+RUN yum install -y \
+    bash \
+    ca-certificates \
+    cmake \
+    gcc \
+    git \
+    glibc-static \
+    libselinux-devel \
+    libgcc-6-dev \
+    libltdl-dev \
```

Comment:
I am pretty sure there are no such packages in centos.

Reply:
i think what's needed for centos is this:
```
make
cmake
gcc
git
glibc-static
libtool-ltdl-devel
libseccomp-devel
device-mapper-devel
btrfs-progs-devel
```

Reply:
Removing:
```diff
-    libselinux-devel \
-    libgcc-6-dev \
-    libltdl-dev \
```

Reply:
also, glibc-static is not needed as we're doing a dynamic build